lib/mayaUsd/python/wrapUtil.cpp

```diff
@@ -56,5 +64,6 @@ void wrapUtil()
                   .def(
                       "getPathRelativeToMayaSceneFile",
                       UsdMayaUtilFileSystem::getPathRelativeToMayaSceneFile)
+                  .def("ensureUSDFileExtension", ensureUSDFileExtension)
```

Comment:
Expose a new python helper. Needed helper function with return value (as opposed to output variable) to call here.